src/selectors/makeGetProjectPreview.js

```diff
@@ -5,6 +5,11 @@ import {generateTextPreview} from '../util/compileProject';
 export default function makeGetProjectPreview() {
   return createSelector(
     (state, {projectKey}) => state.getIn(['projects', projectKey]),
-    project => generateTextPreview(project.toJS()),
+    (project) => {
+      if (project) {
```

Comment:
When do we need this check?

Reply:
Seems to be catching on project load but haven't tracked it down entirely